network/benchmark/src/lib.rs

```diff
@@ -0,0 +1,605 @@
+// Copyright © Aptos Foundation
```

Comment:
nit: full license?
```
// Copyright © Aptos Foundation
// SPDX-License-Identifier: Apache-2.0
```

Reply:
I'm surprised we don't have a lint check for that

Reply:
Yeah, the lint is in a bad state because we can't check for two patterns 😵‍💫